src/lib/permissions/index.ts

```diff
@@ -2,4 +2,5 @@ export * from './interfaces';
 export * from './hasAccessToSpace';
 export * from './clients';
 export * from './forums';
+export * from './proposals';
```

Comment:
each of these should probably be namespaced as well

Reply:
Saw the PR in core, I like that pattern alot. I'd be +1 to have less top-level core and more specific exports, kinda like how we use Mui Material subpaths (without that extreme of granularity)

added @ charmverse/core/tests here for our utils